content/software/ide-v2/tutorials/getting-started/01.ide-v2-downloading-and-installing/ide-v2-downloading-and-installing.md

```diff
@@ -17,7 +17,7 @@ You can easily download the editor from the [Arduino Software page](https://www.
 
 - **Windows** - Win 10 and newer, 64 bits
 - **Linux** - 64 bits
-- **Mac OS X** - Version 10.14: "Mojave" or newer, 64 bits
+- **Mac OS X** - Version 10.15: "Catalina" or newer, 64 bits
```

Comment:
```suggestion
- **macOS** - Version 10.15: "Catalina" or newer, 64 bits
```

Reply:
macOS version after 10.12 should be like this and not Mac OS X ⭐ 

as we update this sentence, we might as well update the naming as well 👍🏼